src/it/resources-noncompilable/com/google/checkstyle/test/chapter3filestructure/rule3421overloadsplit/InputConstructorsDeclarationGroupingRecords.java

```diff
@@ -0,0 +1,76 @@
+/*
+ConstructorsDeclarationGrouping
+
```

Comment:
What is this for ? 
Please remove, config is in google-checks.xml

Reply:
Done. I copied the test cases from the module's actual tests and forgot to remove it